qiskit/transpiler/passes/layout/vf2_layout.py

```diff
@@ -80,18 +82,31 @@ def __init__(
                 based on the number of edges in the interaction graph or the coupling graph
                 (whichever is larger). If set to a value <= 0 no limit on the number of trials
                 will be set.
+            target (Target): A target representing the backend device to run vf2layout on.
+                If specified it will supersede a set value for ``properties`` and
+                ``coupling_map``.
+
+        Raises:
+            TypeError: If neither ``coupling_map`` or ``target`` are provided.
```

Comment:
`__init__` doesn't raise if neither are provided but `run` does - perhaps we should be clearer:
```suggestion
        Raises:
            TypeError: At runtime, if neither ``coupling_map`` or ``target`` are provided.
```